rqd/rqd/rqconstants.py

```diff
@@ -70,6 +70,9 @@
 # Use the PATH environment variable from the RQD host.
 RQD_USE_PATH_ENV_VAR = False
 
+# Use the environment variables from the RQD host.
```

Comment:
I think this requires a more detailed explanation. In the future when we don't have the scope in which this was implemented, it will be hard to understand what this is about. 

Beside that, your implementation is too broad, it could catch any envvar that contains `PATH`, when what you really want is to change `PATH` and `PYTHONPATH`.

Since there's already a option for `PATH` (`RQD_USE_PATH_ENV_VAR`), I suggest you add an option specifically for `PYTHONPATH` -> `RQD_USE_PYTHONPATH_ENV_VAR`.

Reply:
I understand your point and it was one concern I had. 
In fact, the need is wider than only changing `PYTHONPATH`. I need to set env vars when starting up the RQD host, for example for a root path like `MYPROJ_RENDER_ROOT_PATH` which would be set different depending on the artist machine (`Z:` for a windows, `/mnt/myproj/renders` for a linux...). That's why I've proposed something that broad, because able to deal with all env var use cases.

I could add a dedicated `RQD_USE_PYTHONPATH_ENV_VAR` if it is better to split, but we should discuss about a way to set any env var we like when starting up a RQD host.

Reply:
It could be solved by the feature introduced in this PR https://github.com/AcademySoftwareFoundation/OpenCue/pull/1270

Reply:
I've studied the change proposed in this PR https://github.com/AcademySoftwareFoundation/OpenCue/pull/1270, and it is a good proposal, but not a full solution for my use case. The env vars I'd like to set to the RQD process from the host must be set on the fly, depending on the list of projects the artist has access to for example, I don't want to keep the config file up-to-date for each computer.
The solution I see could be to add an argument `--host-env-vars MYENVVAR MYPROJ_RENDER_ROOT_PATH ...` to list the env vars I want the RQD process to load from the host.

Reply:
I get your point, and I think both the use case handled by #1270 and this PR are valid and could co-exist. 

My suggestion is that we change the constant name on this PR to `RQD_COPY_ALL_HOST_ENV_VARS` or something similar to avoid confusion with `RQD_HOST_ENV_VARS` defined on #1270 . 

Besides that, please add the new option to the [rqd.conf example file](https://github.com/Wolf-Pipeline/OpenCue/blob/b6dd28bcfc3c264287cba3ae715fb40ff34fec00/rqd/rqd.example.conf).